backend/common/src/main/java/ai/verta/modeldb/common/CommonHibernateUtil.java

```diff
@@ -166,13 +166,20 @@ private String getDatasourceClass(RdbConfig rdbConfiguration) {
     throw new ModelDBException("Unrecognized database " + rdbConfiguration.getRdbDialect());
   }
 
-  public static void changeCharsetToUtf(JdbcConnection jdbcCon)
+  public static void changeCharsetToUtf(JdbcConnection jdbcCon, RdbConfig rdbConfiguration)
       throws DatabaseException, SQLException {
     try (var stmt = jdbcCon.createStatement()) {
       String dbName = jdbcCon.getCatalog();
-      var sql =
-          String.format(
-              "ALTER DATABASE `%s` CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci;", dbName);
+      String sql;
+      if (rdbConfiguration.isMssql()) {
+        sql =
+            String.format(
+                "ALTER DATABASE \"%s\" COLLATE Latin1_General_100_CI_AI_SC_UTF8;", dbName);
+      } else {
+        sql =
+            String.format(
+                "ALTER DATABASE `%s` CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci;", dbName);
```

Comment:
My understanding is that `utf8mb4_unicode_ci` is better to use than `utf8mb4_general_ci`.  I know this code was already using `utf8mb4_general_ci`, and making this change now might cause problems, but I wanted to make a note.

Reply:
makes sence, but the current change was about utf-8 in Microsoft SQL, this line is about mysql